pandas/io/html.py

```diff
@@ -1041,7 +1041,7 @@ def read_html(
 
     See Also
     --------
-    read_csv
+    read_csv: Read a comma-separated values (csv) file into DataFrame.
```

Comment:
``read_csv`` can actually also read various non-comma-separated values by supplying a different value to the ``sep`` parameter. Can you adjust the sentence a bit.